projects/baby_driver/scripts/can_util.py

```diff
@@ -183,3 +183,33 @@ def next_message(
         ))
 
     return msg
+
+def can_pack(data_list):
+    """
+    Converts list of tuples and combines them into a little-endian
+    bytearray rendition. List is traversed in reverse order, each val
+    is broken into bytes and appended to bytearr output (LSB first)
+    Args:
+        List of tuples in form ((int) val, (int) len_in_bytes). val must be
+        in range [0, 2**len_in_bytes - 1]
+    Returns:
+        A bytearray object in little endian format, representing
+        message components input
+    Raises:
+        ValueError: if invalid values for val, len_in_bytes input
+    """
+    bytearr = bytearray()
```

Comment:
One more thing, sorry: could you change `can_pack` to return a list rather than a bytearray? Shouldn't be much of a change other than changing this line to use `[]` rather than `bytearray()` and then changing the documentation. The reason is that `send_message` is actually expecting a list rather than a bytearray, sorry.